apps/web/app/review-modal/components/MediaViewer.tsx

```diff
@@ -0,0 +1,140 @@
+  const swiperRef = useRef<SwiperType | null>(null);
+
+  const [progress, setProgress] = useState(0);
+  const [overlay, setOverlay] = useState<null | 'play' | 'pause'>(null);
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const video = mediaList.find((m) => m.type === 'video');
+  const images = mediaList.filter((m) => m.type === 'image');
+
+  useEffect(() => {
+    const videoEl = videoRef.current;
+    if (!videoEl) return;
+
+    const handleTimeUpdate = () => {
+      setProgress((videoEl.currentTime / videoEl.duration) * 100);
+    };
+
+    videoEl.addEventListener('timeupdate', handleTimeUpdate);
+    return () => {
+      videoEl.removeEventListener('timeupdate', handleTimeUpdate);
+    };
+  }, [video?.url]);
+
+  const handleVideoClick = () => {
+    if (!videoRef.current) return;
+    if (videoRef.current.paused) {
+      videoRef.current.play();
+      setOverlay('play');
+      setTimeout(() => setOverlay(null), 300);
+    } else {
+      videoRef.current.pause();
+      setOverlay('pause');
+    }
+  };
+
+  const navBtnClass =
+    'border-1 size-[3.2rem] -rotate-90 border-gray-200 bg-white';
+
+  return (
+    <div className="relative flex h-full w-full overflow-hidden rounded-l-xl">
+      {video && (
+        <div className="relative h-full w-full">
+          <video
+            ref={videoRef}
+            src={video.url}
+            controls={false}
+            muted
+            loop
+            playsInline
+            className="h-full w-full cursor-pointer object-cover"
+            onClick={handleVideoClick}
+          />
+          <div
+            className={cn(
+              'pointer-events-none absolute inset-0 flex items-center justify-center bg-black/40 transition-opacity',
+              overlay ? 'opacity-100' : 'opacity-0'
+            )}
+          >
+            {overlay === 'play' && (
+              <SvgPlayArrow className="size-[7.2rem] fill-white" />
+            )}
+            {overlay === 'pause' && (
+              <SvgPause className="size-[7.2rem] fill-white" />
+            )}
+          </div>
+          <Progress
+            value={progress}
+            height="0.25rem"
+            className="absolute bottom-2 left-4 right-4 bg-gray-600"
+          />
+        </div>
+      )}
+      {images.length > 0 && (
+        <>
+          <Swiper
+            direction="horizontal"
+            onSwiper={(s) => (swiperRef.current = s)}
+            onSlideChange={(s) => setCurrentIndex(s.activeIndex)}
+            className="absolute inset-0 h-full w-full"
+          >
+            {images.map((img, i) => (
+              <SwiperSlide key={i}>
+                <Image
+                  src={img.url}
+                  alt="리뷰 이미지"
+                  fill
+                  className="relative h-full w-full object-cover"
+                />
+              </SwiperSlide>
+            ))}
+          </Swiper>
+
+          {images.length > 1 && currentIndex > 0 && (
```

Comment:
```suggestion
          {currentIndex > 0 && (
```
 currentIndex > 0 체크 만으로도 충분히 나타내질것 같네요